.gitignore

```diff
@@ -63,9 +66,11 @@ pylint.txt
 _book/
 
 # Local development stuff
+forum/
 dev/
 dev-manage.py
 .vscode/
+env/
```

Comment:
What is `env` directory used for?

Reply:
It's just another alternative for `venv` that apparently some plugin on pycharm makes.

Reply:
Ok